addons/hr_expense/models/hr_expense_sheet.py

```diff
@@ -369,6 +369,11 @@ def _compute_is_editable(self):
         is_hr_admin = self.user_has_groups('hr_expense.group_hr_expense_manager')
         is_approver = self.user_has_groups('hr_expense.group_hr_expense_user')
         for sheet in self:
+            if is_hr_admin and sheet.state in {'draft', 'submit', 'approve'}:
```

Comment:
Could be just:
```suggestion
            if is_hr_admin:
```
after https://github.com/odoo/odoo/blob/140e58c5a68db674fc1d240147e4d669b29f2cad/addons/hr_expense/models/hr_expense_sheet.py#L372-L375

Reply:
Since it was this way I assumed that sheets in a Posted/Done/Canceled cannot be changed by anyone
look here: https://github.com/odoo/odoo/blob/48c118db2128a9bfbef5f30bdaa9f2377f2c9ae7/addons/hr_expense/models/hr_expense_sheet.py#L377

Maybe I should move the hr_admin part below this one  ?

Reply:
Indeed I didn't notice that part, thank you :)